example/menu/gui.gui

```diff
@@ -0,0 +1,248 @@
+script: "/example/menu/gui.gui_script"
+fonts {
+  name: "system_font"
+  font: "/builtins/fonts/system_font.font"
+}
+background_color {
+  x: 0.0
+  y: 0.0
+  z: 0.0
+  w: 0.0
+}
+nodes {
+  position {
+    x: 677.0
+    y: 528.0
+    z: 0.0
+    w: 1.0
+  }
+  rotation {
+    x: 0.0
+    y: 0.0
+    z: 0.0
+    w: 1.0
+  }
+  scale {
+    x: 3.877
```

Comment:
Don't scale the nodes. Change the size instead!

Reply:
You are still scaling the nodes. Please change size instead of scale. With the scale changed the text gets stretched.

Reply:
now i have understood how to set size

Reply:
Change from Size Mode: Auto to Size Mode: Manual on the box node. This will enable you to change the size fields.